multithread_test.py

```diff
@@ -0,0 +1,82 @@
+# Help from : https://pymotw.com/2/threading/, https://www.tutorialspoint.com/python/python_networking.htm, https://github.com/UCLA-CS130/AAAAA/blob/master/test/integration_multithread_test.py
+
+from os import system
+from threading import Thread
+import socket
+import sys
+import subprocess
+import time
+
+PORT = 2020
+BUFFER_LEN = 1024
+HOLD_REQUEST = "GET /hold HTTP/1.1"
+STATIC_REQUEST = "GET /static/test.html HTTP/1.1\r\n\r\n"
+EXPECTED_STATIC_RESPONSE = "HTTP/1.1 200 OK\r\nContent-Type: text/html\r\n\r\n<!DOCTYPE html>\r\n<html>\r\n<head>\r\n<title>Server Test</title>\r\n</head>\r\n<body>\r\n<h1>Testing testing 1 2 3...</h1>\r\n<p>Looks like the server is working!</p>\r\n</body>\r\n</html>"
```

Comment:
the part that i was worried about was that I would still need the header of "HTTP/1.1 200 OK" from the expected response.